WordPress/AbstractFunctionParameterSniff.php

```diff
@@ -77,6 +78,39 @@ public function process_matched_token( $stackPtr, $group_name, $matched_content
 		}
 	}
 
+	/**
+	 * Verify if the current token is a function call. Behaves like the parent method, except that
+	 * it returns false if there is no opening parenthesis after the function name (likely a
+	 * function import) or if it is a first class callable.
+	 *
+	 * @param int $stackPtr The position of the current token in the stack.
+	 *
+	 * @return bool
+	 */
+	public function is_targetted_token( $stackPtr ) {
+		if ( ! parent::is_targetted_token( $stackPtr ) ) {
+			return false;
+		}
+
+		$next = $this->phpcsFile->findNext( Tokens::$emptyTokens, ( $stackPtr + 1 ), null, true );
+
+		// Not a function call (likely a function import).
```

Comment:
Nitpick: the comment as-is, should be _within_ the condition (above the `return`). Otherwise it is confusing.